src/main/java/models/AccountModel.java

```diff
@@ -0,0 +1,32 @@
+package models;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Random;
+
+@Getter
+@Setter
+public class AccountModel {
+    public CustomerModel customerModel;
+    public Boolean isSubscribedToNewsletter;
+    public String loginName;
+    public String password;
+
+    public AccountModel(CustomerModel customerModel, Boolean isSubscribedToNewsletter) {
+        this.customerModel = customerModel;
+        this.isSubscribedToNewsletter = isSubscribedToNewsletter;
+        this.loginName = customerModel.getLogin();
+        this.password = customerModel.getPassword();
+    }
+
+    public AccountModel() {
+        var customerModel = CustomerModelUtils.generateCustomerWithRandomData();
+        this.customerModel = customerModel;
+        this.loginName = customerModel.getLogin();
+        this.password = customerModel.getPassword();
+
+        Random random = new Random();
+        this.isSubscribedToNewsletter = random.nextBoolean();
```

Comment:
I would use false bu default cause every customer is not subscribed at the start of his lifecycle :)

Reply:
Done
```java
    @Builder.Default
    public Boolean isSubscribedToNewsletter = false;
```